src/views/Main/Catalog/Components/EntityTypeList.js

```diff
@@ -60,7 +60,13 @@ export class EntityTypeList extends React.Component {
   }
 
   showError = () => {
-    this.setState({ error: true });
+    this.setState({
```

Comment:
this looks like you're resetting the state to the initial state. if that's the case, then all of the values for the initial state should be some constant somewhere.

```
const INITIAL_STATE = {
  foo: '',
  bar: false
};

...

showError = () => {
  this.setState(INITIAL_STATE);
}
```

this will become second nature once we start doing redux